docs/details/advisors/index.md

```diff
@@ -1 +1,15 @@
-# Advisor Documentation
\ No newline at end of file
+# Advisor Documentation
+
+## [How to develop security checks](./checks/develop-checks.md)
```

Comment:
@catalinaadam I'm not really happy with how this one looks like, but couldn't come up with a good alternative. Maybe you can? :)